e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -243,6 +256,28 @@ describe('I18456 Built-in slash commands: common', () => {
                 });
         });
     });
+
+    it('MM-T658 /invite - current channel', () => {
+        loginAndVisitDefaultChannel(user1, `${team1.name}/channels/${testChannel.name}`);
+
+        // # Post `/invite @username` where username is a user who is not in the current channel
+        cy.postMessage(`/invite @${userGroup[0].username}`);
```

Comment:
nit: Please create variable for active user to invite so it's easier to read.
``const userToInvite = userGroup[0];``